### DISCUSSION TITLE
are nuggets supported in .net maui previews.??

### DISCUSSION THREAD

**Discussion by itsabhinavsharma** [dotnet/maui#3830] [General] [answered]
I was wondering when the nuggets are going to support .NET maui .. as yesterday i was trying to make basic crud application in dot net maui to test preview.... but when I installed SQLite nugget it gave 420 risk warnings and even it didn't installed....

**Comment by chris-ux** [comment]


**Reply by itsabhinavsharma** [reply]
![1](https://user-images.githubusercontent.com/32599498/147033374-b10ad900-d9ac-4ed1-a148-fecd4b57806d.png)

here see he number of warnings!!

**Comment by nogginbox** [comment, marked as answer]
If a nuget package is a generic .net core or .net standard package then you can use any.

If it is a nuget package that depends on any Xamarin packages then it might not work, but the warnings may not be as bad as they sound.

Any Xamarin dependent package will need to make updates to support Maui and you'll need to ask them if and when they have plans to do that. I'm sure many of them will be working on this, or may have preview packages that already support this.

It looks like you can use SqlLite as this [SqlLite on MAUI blog post](https://vladislavantonyuk.azurewebsites.net/articles/Adding-SQLite-to-the-.NET-MAUI-application) describes how to do.
